handlers/aws/handler.py

```diff
@@ -304,21 +316,24 @@ def handle_timeout(
 
             is_continuation_of_cloudwatch_logs = is_continuing_of_cloudwatch_logs(sqs_record)
 
+            input_id = sqs_record["eventSourceARN"]
```

Comment:
just want to make sure we dont need to check if "eventSourceARN" in sqs_record here?

Reply:
`eventSourceARN` is always present in the record: https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html